lib/Ed25519VerificationKey2020.js

```diff
@@ -137,29 +161,33 @@ export class Ed25519VerificationKey2020 extends LDVerifierKeyPair {
   }
 
   /**
-   * Adds a public key base to the key pair object.
-   *
-   * @param {object} options - Options hashmap.
-   * @param {object} options.key - Key pair object being exported.
+   * Exports the serialized representation of the KeyPair
+   * and other information that json-ld Signatures can use to form a proof.
    *
-   * @returns {object} A PublicKeyNode, with key material.
-   */
-  addPublicKey({key}) {
-    key.publicKeyMultibase = this.publicKeyMultibase;
-    return key;
-  }
-
-  /**
-   * Adds the private key material to the key pair object.
-   *
-   * @param {object} options - Options hashmap.
-   * @param {object} options.key - Key pair object being exported.
+   * @param {object} [options={}] - Options hashmap.
+   * @param {boolean} [options.publicKey] - Export public key material?
+   * @param {boolean} [options.privateKey] - Export private key material?
    *
-   * @returns {object} The keyNode with encoded private key material.
+   * @returns {object} A public key object
+   *   information used in verification methods by signatures.
```

Comment:
Seems like some punctuation or something is needed here.

Reply:
Oh huh... yeah. fixing.